test/controllers/user_controller_test.exs

```diff
@@ -0,0 +1,29 @@
+defmodule ResearchResource.UserControllerTest do
+  use ResearchResource.ConnCase
+  alias ResearchResource.Repo
+  alias ResearchResource.User
+
+  test "GET /users/new", %{conn: conn} do
+    conn = get conn, "/users/new"
+    assert html_response(conn, 200) =~ "Create User"
+  end
+
+  test "POST /users/create - success", %{conn: conn} do
+    conn = post conn, user_path(conn, :create), user: %{
+      email: "me@test.com",
+      password: "secret",
+      first_name: "User",
+      last_name: "Test"
+    }
+    user = Repo.get_by!(User, %{email: "me@test.com"})
+    assert user.ttrrid =~ "TTRR"
+    assert String.length(user.ttrrid) == 11
```

Comment:
When we create the id, we're padding it with 0s between the 'TTRR' and the number (the postgres-generated id). This test is just checking it's using the expected amount of 0s. We could probably add in a couple more checks to be a little more thorough